test/test_bundle_utils.py

```diff
@@ -0,0 +1,682 @@
+import contextlib
+# import copy
+import glob
+import json
+import os
+import pytest
+import re
+
+# from collections import namedtuple
```

Comment:
Some commented out imports you may want to clean up

Reply:
I'll do that.